content/blog/2024/10/04/2024-10-04-content-security-policy-grant.adoc

```diff
@@ -0,0 +1,47 @@
+---
+:layout: post
+:title: "Alpha Omega Foundation Content Security Policy Grant"
+:tags:
+- donation
+authors:
+- markewaite
+description: >
+  Alpha Omega Foundation funds 3 months of work on Jenkins Content Security Policy
+links:
+  discourse: https://community.jenkins.io/t/content-security-policy/nnnnn
+opengraph:
+  image: /images/post-images/2024/10/04/content-security-policy-grant.png
+---
+
+Alpha-Omega has provided a grant from for three months of full-time work to improve the link:/doc/developer/security/csp/[Jenkins implementation] of link:https://content-security-policy.com/[Content Security Policy].
+The improvements will be implemented in October, November, and December of 2024.
+link:/blog/authors/shlomomdahan/[Shlomo Dahan] is implementing the improvements with technical guidance from link:/blog/authors/basil/[Basil Crow] and project management support from link:/blog/authors/gounthar/[Bruno Verachten].
+
+link:https://alpha-omega.dev/[Alpha-Omega] is an associated project of the OpenSSF, established in February 2022, funded by Microsoft, Google, and Amazon.
+The mission of Alpha-Omega is protect society by catalyzing sustainable security improvements to the most critical open source software projects and ecosystems.
```

Comment:
I hesitate to apply this suggestion because I copied the text from https://alpha-omega.dev/ and they use "open source" rather than "open-source".  There are 485 lines with "open source" in the jenkins.io docs site and 275 lines with "open-source".  It seems either form is OK on the site and there is a preference towards "open source".

@kmartens27 can you offer guidance here?

Reply:
Sure thing! The hyphen would be used when open source is being used as an adjective/descriptor. So in this case, it is describing the software projects and ecosystems. Therefore, it would be "open-source software projects and ecosystems" here.